From 6398515adfc86813686605019a3e22d49cd95517 Mon Sep 17 00:00:00 2001
From: nobu <nobu@b2dd03c8-39d4-4d8f-98ff-823fe69b080e>
Date: Fri, 19 Jun 2015 06:04:00 +0000
Subject: [PATCH] test_gem_remote_fetcher.rb: get rid of errors

* test/rubygems/test_gem_remote_fetcher.rb (start_ssl_server):
  temporary measure for "dh key too small" error of OpenSSL
  1.0.2c+.

git-svn-id: svn+ssh://ci.ruby-lang.org/ruby/trunk@50972 b2dd03c8-39d4-4d8f-98ff-823fe69b080e
---
 test/rubygems/test_gem_remote_fetcher.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/rubygems/test_gem_remote_fetcher.rb b/test/rubygems/test_gem_remote_fetcher.rb
index 6b29e18..63dd8fe 100644
--- a/test/rubygems/test_gem_remote_fetcher.rb
+++ b/test/rubygems/test_gem_remote_fetcher.rb
@@ -979,7 +979,7 @@
     end
 
     DIR = File.expand_path(File.dirname(__FILE__))
-    DH_PARAM = OpenSSL::PKey::DH.new(128)
+    DH_PARAM = OpenSSL::PKey::DH.new(2048)
 
     def start_ssl_server(config = {})
       null_logger = NilLog.new
-- 
2.4.3

